### PR TITLE
Adding Droid/iOS/Console sample

### PR DESCRIPTION
**Pull request by mcoutanceau**

### CODE REVIEW

**Review by VJubert — COMMENTED** [review]
les resources.designer.cs ne sont pas à commit

File: src/Xmf2.NavigationGraph.iOS/NavigationPresenter.cs [hunk 1]
```diff
@@ -42,7 +42,11 @@ public async Task UpdateNavigation(NavigationOperation<TViewModel> navigationOpe
 				return;
 			}
 
-			List<PushInformation<TViewModel>> controllersToPush = navigationOperation.Pushes.ConvertAll(x => new PushInformation<TViewModel>(_factoryAssociation[x.Screen], x.Instance));
+			List<PushInformation<TViewModel>> controllersToPush = navigationOperation.Pushes.ConvertAll(x =>
+			{
+				var a = new PushInformation<TViewModel>(_factoryAssociation[x.Screen], x.Instance);
```
Comment (VJubert) [inline src/Xmf2.NavigationGraph.iOS/NavigationPresenter.cs]: si ça pouvait être revert

File: src/Xmf2.NavigationGraph.iOS/NavigationStack.cs [hunk 2]
```diff
@@ -16,12 +16,15 @@ public class NavigationStack<TViewModel> where TViewModel : IViewModel
 
 		public void EnsureInitialized(UIWindow window)
 		{
-			//TODO POURRAIT ÊTRE MIEUX FAIT
```
Comment (VJubert) [inline src/Xmf2.NavigationGraph.iOS/NavigationStack.cs]: no way, revert !!!!!!

**Review by VJubert — CHANGES_REQUESTED** [review]

File: src/Xmf2.NavigationGraph.iOS/NavigationStack.cs (hunk 2 above)
Comment (mcoutanceau) [reply src/Xmf2.NavigationGraph.iOS/NavigationStack.cs]: Previous version does not work neither with the sample app, nor with the one I'm working on.

**Review by VJubert — APPROVED** [review]

**Pull request merged**